apps-script/3p-resources/3p-resources.gs

```diff
@@ -35,13 +35,15 @@ function caseLinkPreview(event) {
       .setTitle(`Case ${caseDetails["name"][0]}`);
     const caseDescription = CardService.newTextParagraph()
       .setText(caseDetails["description"][0]);
-
-    // Returns the card.
-    // Uses the text from the card's header for the title of the smart chip.
-    return CardService.newCardBuilder()
+    const card = CardService.newCardBuilder()
       .setHeader(caseHeader)
       .addSection(CardService.newCardSection().addWidget(caseDescription))
       .build();
+
+    // Returns the link preview smart chip with its dedicated title.
+    return CardService.newLinkPreview()
+      .setTitle(`Case ${caseDetails["name"][0]} - caseDetails["description"][0]`)
+      .setPreviewCard(card);
```

Comment:
@PierrickVoulet can we add the setLinkPreviewTitle as an example of that use as well? It may require modifying the sample or the related images in the guide that reference this sample and sync it with alternative runtime samples.

Reply:
I would rather separate the changes and do that in a follow CL.